extensions/reviewed/Gamepads.json

```
@@ -345,6 +345,26 @@
       ],
       "objectGroups": []
     },
+    {
+      "description": "Return the index of the last last gamepad activated.",
```

```suggestion
      "description": "Return the index of the last gamepad activated.",
```

Double **last**.
Also during my tests, the expression returned the last one plugged and the last gamepad used by using it.
I'm afraid that **activated** was understood only as the last gamepad plugged?

---

Maybe I should rename the method to be called "Last gamepad used"? The intention was that this would represent the playerId of the gamepad that mostly recently had a button touched.